.gitignore

```diff
@@ -53,12 +53,6 @@
 # Backup entities generated with doctrine:generate:entities command
 */Entity/*~
 
-### Symfony Patch ###
-/web/bundles/
```

Comment:
To są pliki automatycznie generowane itp. Normalne static są w katalogu Resources przy bundle.